test/Streamly/Test/Data/Fold.hs

```diff
@@ -1,13 +1,15 @@
 module Main (main) where
 
+import Streamly.Test.Common (checkListEqual)
```

Comment:
This import causes CI to fail. Please check.

Reply:
Fixed.